### DISCUSSION TITLE
MissionControl 0.6.2

### DISCUSSION THREAD

**Discussion by ndeadly** [ndeadly/MissionControl#392] [General]
This is the eleventh official release of Mission Control

This is mostly a bugfix release to address the rumble issues that have surfaced since recent Atmosphère releases. Support for the new N64 and Sega Genesis/Megadrive Nintendo Switch Online controllers has also been added. See below for the full changelog.

New users landing on this release page should first check out the readme on the [main project page](https://github.com/ndeadly/MissionControl) for the official project documentation. There you can find installation and usage instructions along with an FAQ section that will answer most of your questions.

***Atmosphère >= 1.2.0 is required to run this release.***

### Important notice for users of Xbox One controllers
It has been confirmed that the new Xbox Insiders controller firmware update will convert your controller to use Bluetooth LE, which is not currently supported. Please do not update if you wish to continue using your controller with Mission Control

### Changelog:
- Added support for new N64 and Sega Genesis/Megadrive Nintendo Switch Online controllers
- Fixed an issue where controllers would vibrate non-stop under certain conditions after connecting to the console
- Fixed an issue where rumble output was missing from certain games
- Fixed an issue where Crash Team Racing Nitro-Fueled and Crash Bandicoot N. Sane Trilogy games would output a constant stream of random rumble data
- Disabled sending SetTsi command to Ouya controller (fixes disconnect issue)
- Changed the order of initialisation commands for Sony controllers to reduce perceived delay when connecting to the console
- Various improvements to the accuracy of controller subcommand handling were added

<hr /><em>This discussion was created from the release <a href='https://github.com/ndeadly/MissionControl/releases/tag/v0.6.2'>MissionControl 0.6.2</a>.</em>

**Comment by Pierocubo** [comment]
Hi! I have installed this new version and now I can't calibrate the joystick of any controller I have (wiiU/bigbigwon), I press L3 or R3 in the menu and there is no response. Help? I am in the last version of everything. Thanks!

**Reply by ozzilost** [reply]
i can't connect "wiiu pro" controller in this new version i had to go back to the previous version.

**Reply by ndeadly** [reply]
@Pierocubo no problems calibrating for me. Since 13.0.0 you have to tilt and hold the joysticks instead of L3/R3 to begin calibration

@ozzilost my WiiU Pro controllers pair just fine with latest MC/HOS. There have been a few isolated incidents where they didn't seem to want to pair with the console, but this was resolved by a reboot.

**Reply by Pierocubo** [reply]
Thanks for answering, I will try and let you know if it works, thank you very much again!

**Reply by Pierocubo** [reply]
I checked and it is working, thank you very much again.

**Reply by ozzilost** [reply]
> 

it worked after I restart recognized all the controls, thanks.

**Comment by ziete666** [comment]
Hello!!, the rumble issue is still present in my switch, i delete all bt entries then pair it again. as a solo controller there is no problem, the moment i connect another ds4 or any joycon it starts vibrating non stop.

**Reply by ndeadly** [reply]
Are you definitely running the latest version? Please download the 0.6.2 release from GitHub and reinstall (and reboot) just to be sure. Since making the changes in this release I haven't been able to reproduce the issue with any controller.

**Reply by ziete666** [reply]
yes, 1st i updated via HB App Store, since the problem was persistent, i delete it and proceeded to install it manually, again deleting bt entries and all. Sometimes it just stays quiet (no rumble), sometimes not, i play smash and minecraft dungeon.
Version 12.01|AMS M.19.1|E

**Reply by ndeadly** [reply]
Something is not right. I don't think the console should even boot with 0.6.0 or newer installed if you're using the version of Atmosphere you listed. Update Atmosphere before trying anything else. There's no reason not to be up to date.

**Comment by Anish5150** [comment]
I am going to Update my Switch to Firmware 13.1.0 & Atmosphere (v1.2.4) will MissionControl 0.6.2 work on those Firmware and Atmosphere versions. I am new to this Scene thats why I am asking. Thank You

**Reply by ndeadly** [reply]
Yep, works fine on the latest versions

**Comment by shawn8888** [comment]
Quick question: how can I check the version of MissionControl installed on my switch?
It's not listed on hbmenu (through Album) and it's not listed in Data Management -> Manage Software either.

**Reply by Masamune3210** [reply]
It wont be listed in Software because its not an installed app, and it wont be in hbmenu because its a plugin, not a homebrew elf. Best thing to do is just update to latest if you are unsure

**Reply by ndeadly** [reply]
There is a build revision string embedded into the binary, but it is currently unused (and may even be optimised away by the compiler). The intent is to read it out via IPC in the future so it can be easily checked via a gui app.

It might be possible to find this string manually by unpacking `exefs.nsp` from the module directory (it's in the form `<version_number>-<branch_name>-<commit_hash>`, eg. `0.6.2-master-33201eb`) and looking at it in a hex editor, but at this point you're probably better off just (re)installing the latest release from github. There's never any reason not to be on the latest.
